project-metrics/metrics_service/database/models.py

```diff
@@ -196,7 +196,7 @@ def __repr__(self) -> Text:
 
 
 class Cherrypick(Base):
-  """A cherry-picked commit."""
+  """[DEPRECATED] A cherry-picked commit."""
```

Comment:
Add an explanation why, and what replaces it

Reply:
Done